lib/caasp.pm

```diff
@@ -20,11 +20,47 @@ use version_utils 'is_caasp';
 use utils qw(power_action assert_shutdown_and_restore_system);
 
 our @EXPORT = qw(
-  trup_call trup_install rpmver process_reboot check_reboot_changes microos_login
+  trup_call trup_install rpmver process_reboot check_reboot_changes microos_login send_alt
   handle_simple_pw export_cluster_logs script_retry script_run0 script_assert0
   get_delayed update_scheduled
   pause_until unpause);
 
+# Shorcuts for gui / textmode (optional) oci installer
+my %keys = (
```

Comment:
Like this? I think we will get SLE-15 installer soon, so this might be temporary.

Reply:
For example, or inline in the hash:
```perl
my %keys = (
    keyboard     =>is_caasp '4.0+' ? ['e', 'y'] : ['something else'], # VIDEOMODE check can also be injected here so we could skip using arrays
```

Looks like because of this change now we have same keys for both versions, and they were different before.

Reply:
I tried inlines in hash, but code seemd messy with `textmode` & `caasp 3&4` combinations

Reply:
No probs, let's leave it for later. Bus as of now we redefine keyboard, password and ntp keys to the same values, I guess this is not intended.